### PR TITLE
Implement query parameter for assessment

### LINKED ISSUE #31: Implement available query parameters of the current data model
The new API should provide query parameters for the variables our data model currently supports. This Epic will track the progress towards implementing each query parameter.

- [x] https://github.com/neurobagel/bagelapi/issues/26
- [x] https://github.com/neurobagel/bagelapi/issues/15
- [x] #32
- [x] https://github.com/neurobagel/bagelapi/issues/23
- [x] #33
- [x] #34

### LINKED ISSUE #33: Implement cognitive assessment tools as a query parameter
- [x] Implement cognitive assessment in create_query function of utility.py module
- [x] Add cognitive assessment as a field in QueryModel of models.py module
- [x] Update get_query path operation of query.py router and get function of crud.py module to reflect the addition of cognitive assessment
- [x] test parameter implementation/functionality

### PR DESCRIPTION
**Pull request by alyssadai**
The main changes are:

- New optional query parameter added: assessment (regex str)
- Created pytest fixture for mock_invalid_get function
- Added tests for valid and invalid assessment inputs

Closes #33
Closes #31

### CODE REVIEW

File: tests/test_query.py [hunk 1]
```diff
@@ -355,21 +314,12 @@ async def mock_get(
 @pytest.mark.parametrize(
     "invalid_image_modal", ["2nim:EEG", "apple", "some_thing:cool"]
 )
-def test_get_invalid_image_modal(test_app, invalid_image_modal, monkeypatch):
+def test_get_invalid_image_modal(
+    test_app, mock_invalid_get, invalid_image_modal, monkeypatch
```
Comment (surchs) [inline tests/test_query.py:318]: looks good, but we should try to have this kind of whitespace formatting be done as part of the linting so it'll be consistent. Could you start an issue @alyssadai to discuss what rules we want to have for the code formatting and how we enforce them?

**Review by surchs — APPROVED** [review]
Thanks for the PR @alyssadai . This all looks great to me :tada:. Good to go

**Comment by surchs** [comment]
also thanks for getting the coverage reports to post in the PR! Looks very nice

**Review by rmanaem — APPROVED** [review]
Nicely done, looks good to me!

File: tests/test_query.py (hunk 1 above)
Comment (alyssadai) [reply tests/test_query.py:318]: Aha, this formatting was actually applied by the linter! By the pre-commit hook for black specifically. At the moment, when a line of code surpasses a certain character count (I believe we set it to the PEP8 standard) it gets automatically reformatted like this prior to commit.

**Pull request merged**